tripal_chado/config/install/tripal.tripalfield_collection.general_chado.yml

```
@@ -1514,6 +1514,33 @@ fields:
               region: content
               weight: 12
 
+    -   name: 'pub_publication_type'
+        content_type: 'pub'
+        label: 'Additional Publication Type'
```

Since this is an unlimited field, shouldn't it be optionally plural?
```suggestion
        label: 'Additional Publication Type(s)'
```